ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -101,6 +102,15 @@ def pytest_configure(config):
     """
     if not (config.getoption("--help") or config.getoption("collectonly")):
         process_cluster_cli_params(config)
+        config_file = os.path.join(
+            ocsci_config.RUN['log_dir'],
+            f"run-{ocsci_config.RUN['run_id']}-config.yaml"
+        )
+        print_cofig_to_file(config_file)
+        log.info(
+            f"Dump of the consolidated config file is located here: "
+            f"{config_file}"
```

Comment:
Writing the config contents to a file and then logging the filename doesn't seem like it'll be enough; most of us don't have access to the filesystem your builds are using.

Reply:
If it will be in the `log_dir`, it should be accessible on the common NFS share available via http and linked in each build.
We can also consider to print it to the console in Jenkins job, if it will be helpful.